### ISSUE TITLE
Listing Page Spacing Issue

### ISSUE THREAD

**Issue by bamboochow** [whynotearth/hotel-website-vkirirom.com#281]
### Bug
For ALL non-blog listing pages, there is a large space between the page title and the descriptions. This only shows up for desktop and tablet, mobile currently has no spacing issue. 
![image](https://user-images.githubusercontent.com/14948326/71707978-68c82600-2e20-11ea-90e2-00ffaca727a2.png)
![image](https://user-images.githubusercontent.com/14948326/71708100-60241f80-2e21-11ea-9a1b-2d7ce658d326.png)

Issue does not happen in any blog page related pages 
https://develop.vkirirom.com/listing/Great-Photo-Opportunities
https://develop.vkirirom.com/listing/First-Ever-Progressive-Web-Application-in-Cambodia

### Fix 
Remove the spacing issue here so that there isn't anymore large empty spaces on listing pages 
![image](https://user-images.githubusercontent.com/14948326/71708177-dde82b00-2e21-11ea-861c-be3250ba02c5.png)

### Link (some examples)
https://develop.vkirirom.com/listing/Team-Building-Activities
https://develop.vkirirom.com/listing/Luxury-Tent
https://develop.vkirirom.com/listing/Pine-View-Kitchen-PVK
https://deploy-preview-247--cranky-nightingale-3731fc.netlify.com/listing/Pine-View-Kitchen-PVK

**Comment by mort3za** [comment]
Can't see the bug, probably it's resolved by my previous changes.

**Comment by bamboochow** [comment]
ok, I'm going to keep this on for a while so I keep an eye on it for a bit in case it pops up, will close once everything is merged into develop
